src/components/sections/preface/listing/background/index.js

```diff
@@ -0,0 +1,120 @@
+import * as THREE from 'three'
+
+var mouseX = 0,
+  mouseY = 0,
+  windowHalfX = window.innerWidth / 2,
+  windowHalfY = window.innerHeight / 2,
+  SEPARATION = 200,
+  AMOUNTX = 10,
+  AMOUNTY = 10,
+  camera,
+  scene,
+  renderer
+
+// init();
+// animate();
+
+function onWindowResize() {
+  windowHalfX = window.innerWidth / 2
```

Comment:
There will be an error in building process, because window doesnt exist in prerender process

Reply:
fixed